manylinux2014-wheel-build/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM quay.io/pypa/manylinux2014_x86_64:latest
+
+run mkdir /src && mkdir /Pillow
+ENV SRC=/src
+ENV PATH=/opt/python/cp27-cp27m/bin:/opt/rh/devtoolset-2/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
Do we need cp27m stuff here or can we use something newer?

Reply:
Turns out no, it isn't needed - I've pushed a commit and the job passes.